boefjes/boefjes/plugins/kat_kat_finding_types/kat_finding_types.json

```diff
@@ -21,14 +21,14 @@
     "recommendation": "This header is not supported by default by Mozilla. If this header is required for your environment: Set the HTTP header X-Permitted-Cross- Domain-Policies: none in all HTTP responses. Use value master-only if a Flash or Acrobat cross- domain configuration file is used that is placed in the root of the web server"
   },
   "KAT-NO-EXPLICIT-XSS-PROTECTION": {
```

Comment:
Should we invert the name of this finding?
It seems the finding states to Remove the header?